libraries/AP_InertialSensor/AP_InertialSensor_LSM9DS0.cpp

```diff
@@ -468,6 +472,10 @@ bool AP_InertialSensor_LSM9DS0::_hardware_init()
         goto fail_whoami;
     }
 
+    // setup for register checking
+    _dev_gyro->setup_checked_registers(6);
+    _dev_accel->setup_checked_registers(5);
```

Comment:
Have I counted wrong or is there a reason for these to have one more register than those marked as checked?